config/src/config/consensus_config.rs

```diff
@@ -202,7 +202,7 @@ impl Default for ConsensusConfig {
                 PipelineBackpressureValues {
                     back_pressure_pipeline_latency_limit_ms: 2500,
                     max_sending_block_txns_override: 2000,
-                    max_sending_block_bytes_override: 1024 * 1024,
+                    max_sending_block_bytes_override: 1024 * 1024 + BATCH_PADDING_BYTES as u64,
```

Comment:
we are adding these, so that we can still execute 1MB transactions, correct?

or is there a different reason?

Reply:
Yeah, I've changed the batch size to allow 1MB transaction, so need to change the smallest block size as well.